lib/flay.rb

```diff
@@ -63,7 +63,7 @@ def self.parse_options args = ARGV
 
     OptionParser.new do |opts|
       opts.banner  = "flay [options] files_or_dirs"
-      opts.version = Flay::VERSION
+      opts.version = VERSION
```

Comment:
the subclass overwrites the default options and some methods ... does not need the VERSION, but it is theoretically wrong to hardcode the class so I'm removing all the hardcoded classes so nobody has to come back here and to make it consistent